wp-content/themes/missilethreat/assets/_scss/components/_system-elements.scss

```diff
@@ -3,6 +3,23 @@
 .system-elements {
   margin-bottom: rem(32px);
 
+  @include breakpoint('small') {
+    max-width: 100vw;
+  }
+
+  @include breakpoint('medium') {
+    float: unset !important;
```

Comment:
Why do you need the `important` selectors here?

Reply:
There's a style (from what I can tell it's a default WP style) being added to images that is forcing me to use `!important` in my alignments, which in turn made it so I had to use it on the system elements.

Reply:
Huh, that's weird. Good to know!